### PR TITLE
Bridge hardening

### PR DESCRIPTION
**Pull request by VladimirNovgorodov**
Added a special user to run the Docker. 
The private key is put in /root/.env with read-write only for root. 
The key is passed as a parameter to  signature-request watcher only.
Validator address get via PrivateKeyToAddress.js.

### CODE REVIEW

File: bridge-nodejs/roles/servinstall/templates/poabridge.j2 [hunk 1]
```diff
@@ -10,45 +10,63 @@
...
+done < $keyfile
 
 runbridge(){
     cd $WORKDIR
-    sudo /usr/local/bin/docker-compose run --detach bridge npm run watcher:signature-request
-    sudo /usr/local/bin/docker-compose run --detach bridge npm run watcher:collected-signatures
-    sudo /usr/local/bin/docker-compose run --detach bridge npm run watcher:affirmation-request
-    sudo /usr/local/bin/docker-compose run --detach bridge npm run sender:home
-    sudo /usr/local/bin/docker-compose run --detach bridge npm run sender:foreign
+    sudo -u "{{ compose_service_user }}" /usr/local/bin/docker-compose run -e VALIDATOR_ADDRESS=$vaddr -e VALIDATOR_ADDRESS_PRIVATE_KEY=$vkey --detach bridge npm run watcher:signature-request
```
Comment (akolotov) [inline bridge-nodejs/roles/servinstall/templates/poabridge.j2]: Will the value be logged somewhere (linux logs or the list of running processes) when the particular container is running?

Comment (VladimirNovgorodov) [reply bridge-nodejs/roles/servinstall/templates/poabridge.j2]: Yes. The "logging" ansible role is adding logging to docker-compose file. After that every containers events save to /var/syslog file.

Comment (akolotov) [reply bridge-nodejs/roles/servinstall/templates/poabridge.j2]: Yes, please. I am worrying that it could expose the validator private key.

Comment (VladimirNovgorodov) [reply bridge-nodejs/roles/servinstall/templates/poabridge.j2]: Ok. I am working on it.

Comment (VladimirNovgorodov) [reply bridge-nodejs/roles/servinstall/templates/poabridge.j2]: I added the option to rsyslog.com which will filter out all messages that include "VALIDATOR_ADDRESS_PRIVATE_KEY". So private key wouldn't be contained in any log files.

**Comment by phahulin** [comment]
This is not the last PR related to security audit

**Pull request merged**